lib/core/src/Cardano/Wallet/DB/Sqlite/TH.hs

```diff
@@ -90,7 +92,7 @@ TxMeta
     txMetaWalletId     W.WalletId   sql=wallet_id
     txMetaStatus       W.TxStatus   sql=status
     txMetaDirection    W.Direction  sql=direction
-    txMetaSlot         W.SlotId     sql=slot
+    txMetaSlot         SlotNo       sql=slot
```

Comment:
I suspect we may need manual migration

Reply:
Don't fret over DB migration if it's too hard. We can start from a fresh state directory if necessary.

Reply:
On second thought, I fear/wonder if users would silently get corrupted data, as the meaning of the `slot` table has changed, but not the name.

Is there a good way to test / detect this?

Reply:
If the type of the column changes, I think persistent auto migrations will either throw an exception or helpfully delete the table.